packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -36,7 +36,7 @@ const propTypes = {
    */
   endAccessory: PropTypes.node,
   /**
-   * An array of react display elements to be presented.
+   * An array of terra-clinical-item-display's to be presented.
    */
   displays: PropTypes.arrayOf(PropTypes.element),
```

Comment:
if that's the case, should we scope prop type validation to the component too?
```suggestion
   displays: PropTypes.arrayOf(PropTypes.instanceOf(ItemDisplay),
```